src/lando/treestatus/views.py

```diff
@@ -193,352 +329,292 @@ def update_tree_log(
-
-        db.session.add(status_change)
-
-    db.session.commit()
 
     return [
         {
             "tree": tree.tree,
-            "status": new_status.value,
-            "reason": new_reason,
-            "message_of_the_day": new_motd,
+            "status": status.value if status else tree.status,
+            "reason": reason,
+            "message_of_the_day": message_of_the_day,
         }
-        for tree in trees
-    ], 200
+        for tree in combined_trees
+    ]
 
 
+@treestatus_api.get("/trees/{tree}", response={200: Result[TreeData]})
 @result_object_wrap
-def get_tree(tree: str) -> dict:
+def api_get_tree(request: WSGIRequest, tree: str) -> dict:
     """Handler for `GET /trees/{tree}`."""
     result = get_tree_by_name(tree)
     if result is None:
         raise ProblemException(
-            404,
-            f"No tree {tree} found.",
-            "The tree does not exist.",
+            status=404,
+            title=f"No tree {tree} found.",
+            detail="The tree does not exist.",
             type="https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404",
         )
     return result.to_dict()
 
 
-@auth.require_auth0(
-    groups=(auth.TREESTATUS_ADMIN,), scopes=("lando", "profile", "email"), userinfo=True
-)
-def make_tree(tree: str, body: dict):  # noqa: ANN201
-    """Handler for `PUT /trees/{tree}`."""
-    if body["tree"] != tree:
-        raise ProblemException(
-            400,
-            f"Tree names must match, {tree} from url and {body['tree']} from body do not.",
-            "Tree names must match.",
-            type="https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400",
-        )
-
-    new_tree = Tree(
-        tree=tree,
-        status=TreeStatus(body["status"]),
-        reason=body["reason"],
-        message_of_the_day=body["message_of_the_day"],
-        category=TreeCategory(body["category"]),
-    )
+def create_new_tree(
+    user_id: str,
+    tree: str,
+    status: TreeStatus = TreeStatus.OPEN,
+    reason: str = "Initial tree creation",
+    message_of_the_day: str = "",
+    category: TreeCategory = TreeCategory.OTHER,
+) -> Tree:
+    """Create a new `Tree` with the given fields."""
     try:
-        db.session.add(new_tree)
-        db.session.commit()
-    except (sa.exc.IntegrityError, sa.exc.ProgrammingError):
+        new_tree = Tree.objects.create(
+            tree=tree,
+            status=status,
+            reason=reason,
+            message_of_the_day=message_of_the_day,
+            category=category,
+        )
+    except IntegrityError:
         raise ProblemException(
-            400,
-            f"Tree {tree} already exists.",
-            "Tree already exists.",
+            status=400,
+            detail=f"Tree {tree} already exists.",
+            title=f"Tree {tree} already exists.",
             type="https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400",
         )
```

Comment:
alternatively/in addition:

```suggestion
        ) from exc
```
so we can track the exception chains